pages/Configuring/Expanding-functionality.md

```diff
@@ -12,8 +12,8 @@ to react to different events. See its description
 This bash script will change the outer gaps to 20 if the currently focused
 monitor is DP-1, and 30 otherwise.
 
-```sh
-#!/bin/sh
+```bash
+#!/bin/bash
```

Comment:
why are we doing bash here? some people don't use bash

Reply:
Substring expansion `${1:0:10}` is a Bash-only feature

Reply:
zsh handles it well too

Reply:
I have /bin/sh symlinked to /bin/dash so I won't be able to run the script if it uses non-POSIX features despite advertising itself as POSIX shell script with `#!/bin/sh`

Reply:
technically this is only an example and not a production script, eh, whatever